n2kMapper.js

```diff
@@ -81,30 +84,36 @@ var toValuesArray = function (theMappings, n2k, state) {
           return false
         }
       })
-      .map(function (theMapping) {
+      .reduce(function (updates, theMapping) {
```

Comment:
Could the reducer be a static function so that we would not need to create one for each incoming pgn?